operational/azure/total_instance_hours/azure_usage_instance_hours_used.pt

```diff
@@ -0,0 +1,412 @@
+name "Azure Usage Report - Number of Instance Hours Used"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy produces a usage report showing the number of Hours used for each Azure Instance Family. See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/azure/total_instance_hours) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "low"
+category "Operational"
+default_frequency "monthly"
+info(
+  version: "1.0",
+  provider: "Azure",
+  service: "Compute",
+  policy_set: "Usage Report"
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  category "Policy Settings"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+  default []
+end
+
+parameter "param_region" do
```

Comment:
Should this be expanded to let the user specify multiple regions? In principle, nothing would stop implementing that (can just loop through the list to add to the filters in the Optima request JS), and it would add functionality and make the policy more similar to how other policies work.

Reply:
Policy now supports Allow/Deny Filter for Regions. Retested in customer tenant e.g., no filter; filter to only include US East and US West; filter to exclude EU West etc. 

Policy appears to be working producing the chart as expected.